src/Microsoft.MobileBlazorBindings.WebView.macOS/build/Microsoft.MobileBlazorBindings.WebView.macOS.targets

```diff
@@ -0,0 +1,23 @@
+﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk.Razor"/>
+  <Target Name="_StaticWebAssetsComputeFilesToBuild" AfterTargets="GetCopyToOutputDirectoryItems" DependsOnTargets="ResolveStaticWebAssetsInputs">
+    <ItemGroup>
+       <_ExternalBuildStaticWebAsset Include="%(StaticWebAsset.FullPath)" Condition="'%(SourceType)' != ''">
+        <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        <TargetPath>$([MSBuild]::MakeRelative('$(MSBuildProjectDirectory)','$([MSBuild]::NormalizePath('Resources\wwwroot\%(BasePath)\%(RelativePath)'))'))</TargetPath>
+      </_ExternalBuildStaticWebAsset>
+      <_SourceItemsToCopyToOutputDirectory Remove="@(_ExternalBuildStaticWebAsset)" />
+      <_SourceItemsToCopyToOutputDirectory Include="@(_ExternalBuildStaticWebAsset)" />
+    </ItemGroup>
+  </Target>
+  <Target Name="_StaticWebAssetsComputeFilesToBundle" AfterTargets="_DetectSigningIdentity" DependsOnTargets="ResolveStaticWebAssetsInputs">
```

Comment:
The `AfterTargets` on this seems kind of random. Is it based on trying to find out where to "insert" this target based on various other steps, and this happens to be a good spot? For the sake of MBB previews perhaps this is fine, but I do get worried about depending on other SDKs "random" targets (especially ones that start with `_`). Or is this something else?